### DISCUSSION TITLE
"Select Keys" commands should work from selected cells

### DISCUSSION THREAD

**Discussion by Al532** [opentoonz/opentoonz#6808] [Ideas]
I was looking to configure a shortcut to easily select keys in a group of frames. I didn't find something useful.

In the "Configure Shortcuts" menu, a lot of the "Select Keys" commands seem pointless. 
The ones with "Frame" (as Select All Keys in this Frame) are pointless because, indeed, for the command to work, a key has to be selected: hence those commands do nothing at all (than selecting the key that's... already selected!). By the way, what does mean "Select Previous Keys in this Frame", since there's one unique key in a frame? That seems nonsense...

**In fact, all those "...Frame" commands would be useful, if there were usable when selecting a cell, or a group of cells (instead of a key).**

**Comment by RattusaurusRex** [comment]
Select Previous Keys in this Frame will select all keys to the left in frame x.  Here, it's selected all previous keys in frame 6:
![image](https://user-images.githubusercontent.com/60277069/115675036-1788c200-a303-11eb-97fe-a747219a3469.png)

**Comment by Al532** [comment]
So "Previous" means horizontal in that case, that's confusing! But that makes sense, thanks!

However, it would be great to have a command to select the keys in already selected cells, while deselecting the frame.
